src/server/render.ls

```diff
@@ -5,16 +5,11 @@ map = prelude.Obj.map;
 
 __template = jade.compile-file (path.join __dirname, 'index.jade')
 
-exports.escape-state = -> xss-filters.inHTMLData it;
-
-escape-filter = (k, v) -> if typeof v === 'string' then return exports.escape-state v else return v
-
-exports.stringify-state = -> JSON.stringify it, escape-filter
+exports.stringify-state = -> it |> JSON.stringify |> xss-filters.inHTMLData
```

Comment:
Am I reading it right that this is just for tests? Can we not move that function, and possibly even the pipeline in `application.ls` (wrapped in a function) to it's own module? Something like `src/state-serialisation.ls` or similar.